src/Avalanche/Credential.h

```diff
@@ -0,0 +1,45 @@
+// Copyright © 2017-2021 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+#include "../Data.h"
+
+
+namespace TW::Avalanche {
+
+    enum CredentialTypeID {
+      SECPCred = 9,
+      NFTCred = 14
+    };
+    
+    class Credential {
```

Comment:
is `Credential` used in avalanche code base? it looks just signature type and data?

Reply:
Yes, it's the way signature data is represented in a signed transaction: https://docs.avax.network/build/references/avm-transaction-serialization#credentials